tests/Unit/Core/Foundation/VersionTest.php

```diff
@@ -512,17 +512,31 @@ public function provideVersions()
             '8.1.2' => [
                 '8.1.2',
                 [
-                    'version' => '1.8.1.2',
-                    'fullVersion' => '1.8.1.2',
+                    'version' => '8.1.2',
```

Comment:
we should keep legacy behavior too, wdyt? :thinking:

Reply:
this is where the bug is :D

8+ versions should not be converted to 1.8+

Reply:
We should keep the behavior for `1.8.0.1` and `8.0.1` which are different versions

Reply:
I added a test for 1.8+